.github/workflows/draft.yml

```diff
@@ -0,0 +1,44 @@
+name: Release package to Github
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build-binary:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: addnab/docker-run-action@v3
+        with:
+          image: utdemir/ghc-musl:v24-ghc922
+          options: -v ${{ github.workspace }}:/mnt
+          run: |
+            cd /mnt
+            sh build.static.sh
+
+      - name: Change owner before compression
+        run: sudo chown $USER:$USER atomic-write
```

Comment:
@Alex0jk atomic-write doesn't provide an executable. It's just a [library](https://github.com/stackbuilders/atomic-write/blob/master/atomic-write.cabal#L52). We don't need any step related to uploading a binary to a GH release.